docs/blog.md

```diff
@@ -16,32 +17,122 @@ To add blog support to your app, run the following command:
 ```
 
----
+## Add Syntax Highlighting
+
+When you are using code blocks in your markdown files for your blog posts, you'll may define a language for the used code like this:
+
+<pre><code>```ts
+const foo = 'bar';
+```</code></pre>
+
+Scully will parse the markdown using [_marked_](https://www.npmjs.com/package/marked).
+In the end the parsed result will look like this:
+
+```html
+<code class="language-ts">
+  <span class="token keyword">const</span> foo
+  <span class="token operator">=</span>
+  <span class="token string">'bar'</span><span class="token punctuation">;</span>
+</code>
+```
+
+As you can see _marked_ will use the CSS class prefix `language-` for tagging the code block with the appropriate language.
+
+To highlight your code blocks, you can use [_prismjs_](https://prismjs.com) as it matches with the resulting class names. You can install prismjs using NPM and creating a service to highlight your code blocks:
+
+```bash
+npm i --save prismjs
+ng g s highlight
+```
+
+The service will include all languages you need and probably the clipboard library to simply copy the code etc.
 
+It could look like this:
 
+```ts
+import {Injectable, Inject} from '@angular/core';
+
+import {PLATFORM_ID} from '@angular/core';
+import {isPlatformBrowser} from '@angular/common';
+
+import 'clipboard';
+import 'prismjs';
+import 'prismjs/plugins/toolbar/prism-toolbar';
+import 'prismjs/plugins/copy-to-clipboard/prism-copy-to-clipboard';
+import 'prismjs/components/prism-bash';
+import 'prismjs/components/prism-css';
+import 'prismjs/components/prism-javascript';
+import 'prismjs/components/prism-json';
+import 'prismjs/components/prism-markup';
+import 'prismjs/components/prism-typescript';
+// ... probably more, check out node_modules/prismjs/components
+
+declare var Prism: any;
+
+@Injectable()
+export class HighlightService {
+  constructor(@Inject(PLATFORM_ID) private platformId: Object) {}
+
+  highlightAll() {
+    if (isPlatformBrowser(this.platformId)) {
+      Prism.highlightAll();
+    }
+  }
+}
+```
+
+After this, you need the service to be injected in your `BlogComponent` that was generated by scully:
+
+```ts
+import {/* ... */, AfterViewChecked} from '@angular/core';
+import {HighlightService} from '../highlight.service';
+/* ... */
+export class BlogComponent implements OnInit, AfterViewChecked {
+  constructor(
+    /* ... */
+    private highlightService: HighlightService
+  ) {}
+  /* ... */
+  ngAfterViewChecked() {
+    this.highlightService.highlightAll();
+  }
+}
+```
+
+The last step is to include the basic CSS provided by prism and one of the themes:
+
+```css
+@import '~prismjs/plugins/toolbar/prism-toolbar.css';
+/* check node_modules/prismjs/themes/ for the available themes */
+@import '~prismjs/themes/prism-tomorrow';
+```
+
+That's it, all code languages you have been imported in your `HighlightService` should be highlighted now.
+Check out also the official [docs of prismjs](https://prismjs.com/)
+
+---
```

Comment:
@d-koppenhagen 
Could you move the entire section  to other file?. "Add Syntax Highlighting" is not part of the Scully, but is a really interesting sample application of it. 

Let's move all explanation to other file,  `utils.md`  maybe, and linked from ´blog.md´with good paragraph.

Reply:
done, please check it again :)